src/topology/algebra/group.lean

```diff
@@ -28,132 +82,95 @@ class topological_add_group (G : Type u) [topological_space G] [add_group G]
 /-- A topological group is a group in which the multiplication and inversion operations are
 continuous. -/
 @[to_additive]
-class topological_group (G : Type*) [topological_space G] [group G]
+class topological_group (G : Type u) [topological_space G] [group G]
```

Comment:
What's the story behind this change? This isn't needed, right? In that case I think this should be reverted.

Reply:
This was a merge artifact.
```suggestion
class topological_group (G : Type*) [topological_space G] [group G]
```